### DISCUSSION TITLE
milvus.create_index halt after finishing insert vector

### DISCUSSION THREAD

**Discussion by markson14** [milvus-io/milvus#5196] [Q&A and General discussion] [answered]
> ENV
> Machine: (48c) Intel Xeon Patinum 8269 CPU 2.50GHz / 176GB Memory
> total: 10M
> dim: 256
> index_file_size: 2048
> 
> nlist: 4096
> indextype: INV_SQ8

when I run create_index, the process seems halt there for almost half an hour. The usage of CPUs is also very low. When I use keyboard to interupt the process it returns the following image. I'm not sure if it is training or stucking by some Internet issues. Any clues of what's going on here? Thank you.

CPU Usage:
![image](https://user-images.githubusercontent.com/30102000/118073801-28e35c80-b3df-11eb-89b3-6fecbb95a9bd.png)
KeyboardInterrput:
![image](https://user-images.githubusercontent.com/30102000/118073980-a313e100-b3df-11eb-93fd-d4fa65084519.png)

**Comment by shengjun1985** [comment, marked as answer]
> indextype: INV_SQ8

It should be IVF_SQ8
Did you type wrong in the issue, or did you type the command wrong?
And can you provide any error or debug log of the server?

Furthermore, v1.1.0 has been released.
Please upgrade to the latest version to avoid many bugs.

**Reply by markson14** [reply]
logs/milvus-info.log，
![image](https://user-images.githubusercontent.com/30102000/118083210-18d47880-b3f1-11eb-82c6-83e1105fafa5.png)
logs/milvus-debug.log
![image](https://user-images.githubusercontent.com/30102000/118084022-87fe9c80-b3f2-11eb-8f4b-3669b746c549.png)

**Reply by yhmo** [reply]
The log indicate there are 20 segment files waiting to be indexed.
Use this command in terminal to see which file is building index in milvus:
`cat logs/milvus-debug.log |grep 'build index'`

You also can check metadata in MySQL to see how many segment files finish index:
- Firstly login into mysql client:
`mysql -hxxx.xxx.xxx.xxx -uroot -p123456`
- In mysql client, use SQL statement to check metadata:
`use milvus`
`select * from TableFiles;`

**Reply by yhmo** [reply]
"select * from TableFiles" will print all segment files information in milvus. There is a filed named "file_type", if the value=3, means it is an index file; if the value=7, means it is a raw data file and is backuped; if the value=1, means it is raw data file, and it doesn't build index yet.

**Reply by markson14** [reply]
Very detailed. Appreciated!
